engine/renderer/public/pipelines/lighting_pipeline.hpp

```diff
@@ -43,4 +44,5 @@ class LightingPipeline final : public FrameGraphRenderPass
     vk::Pipeline _pipeline;
 
     const BloomSettings& _bloomSettings;
+    const ResourceHandle<GPUImage> _ssaoTarget;
```

Comment:
No need to store this here, as it is only used to get the index to the push constants

Reply:
Removed 🫡